indexer/src/executor.rs

```diff
@@ -209,7 +310,7 @@ mod tests {
                 .expect("Failed to encode"),
         ];
 
-        let result = executor.trigger_event("an_event_name", encoded);
+        let result = executor.trigger_event(ReceiptEvent::Log, encoded);
```

Comment:
Since this was just previously a dummy value, I used `ReceiptEvent::Log` and updated the tests accordingly